package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-social-icons",
   "version": "2.7.0",
   "description": "beautiful, easy svg social icons in react",
-  "main": "./lib/react-social-icons.js",
+  "main": "./src/react-social-icons.js",
```

Comment:
This is not desirable, since on publish to npm we're running the code through babel so that downstream clients don't have to.  This output is in lib and is what `main` should point to.